website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -181,6 +181,24 @@ The `NewControllerManagedBy()` provides a controller builder that allows various
 
 `Owns(&appsv1.Deployment{})` specifies the Deployments type as the secondary resource to watch. For each Deployment type Add/Update/Delete event, the event handler will map each event to a reconcile `Request` for the owner of the Deployment. Which in this case is the Memcached object for which the Deployment was created.
 
+The dependent objects, in this case the Deployments, need to have an [Owner References](https://kubernetes.io/docs/concepts/overview/working-with-objects/owners-dependents/#owner-references-in-object-specifications) field that references their owner object. This will be added by using the method `ctrl.SetControllerReference`. [More info][ks8-doc-owner-ref]
+
+Note: The K8s api will manage the resources according to the`ownerRef` which will be properly set by using this method. Therefore, the K8s API will know that these resources created on the controller, such as the Deployment to run the Memcached Operand image, depend on the custom resource for the Memcached Kind which will also allow the K8s API delete all resources that were created when/if the custom resource itself be deleted. [More info][k8s-doc-deleting-cascate]
```

Comment:
Small nit:

I don't see where the `k8s-doc-deleting-cascate` link reference exists at the end of the document. Make sure to add it at the end of the document similar to the other link references or else we will have a dead link.

Also looks like there may be a typo in the link reference name:
```suggestion
Note: The K8s api will manage the resources according to the`ownerRef` which will be properly set by using this method. Therefore, the K8s API will know that these resources created on the controller, such as the Deployment to run the Memcached Operand image, depend on the custom resource for the Memcached Kind which will also allow the K8s API delete all resources that were created when/if the custom resource itself be deleted. [More info][k8s-doc-deleting-cascade]
```